src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -270,20 +270,6 @@ public override object VisitDynamicType(DynamicTypeSymbol symbol, StringBuilder
                 return null;
             }
 
-            private object VisitTupleType(TupleTypeSymbol symbol, StringBuilder builder)
-            {
-                // get the underlying tuple type. 
-                var tupleType = symbol.UnderlyingNamedType;
-
-                // In the case this was a transformed tuple with unresolved type parameters, get the original defnition so we know about the type parameters
-                if (tupleType.TypeArgumentsNoUseSiteDiagnostics.Any() && tupleType.TypeArgumentsNoUseSiteDiagnostics[0].ContainingSymbol is null)
-                {
-                    tupleType = tupleType.OriginalDefinition;
-                }
-
-                return VisitNamedType(tupleType, builder);
-            }
```

Comment:
If you can recreate the other issue we discussed (a type parameter with a `null` `ContainingSymbol`), please add a test to highlight the problem (`CodeGenTupleTests.cs`). Thanks